2009-11-30  Michael Tautschnig  <mt@debian.org>

	* install_packages: Make aptitude-r method handle MAX_PACKAGES (thanks Stefan
		Goetz). (closes: #558139)
Index: trunk/bin/install_packages
===================================================================
--- trunk.orig/bin/install_packages
+++ trunk/bin/install_packages	
@@ -187,7 +187,7 @@
     next;
   }
 
-  if ($atype eq "install" || $atype eq "smarti" || $atype eq "aptitude" || $atype eq "unpack" || $opt_l || $opt_L) {
+  if ($atype eq "install" || $atype eq "smarti" || $atype eq "aptitude" || $atype eq "aptitude-r" || $atype eq "unpack" || $opt_l || $opt_L) {
 
     mkpackagelist(@{$list{$atype}}); # create lists of known and unknown packages
     if ($opt_l) {
